playbooks/adhoc/s3_registry/s3_registry.yml

```diff
@@ -10,6 +10,9 @@
   remote_user: root
```

Comment:
@sdodson do you agree as well? If so I'll be happy to ``git rm`` this file instead of porting it.